biz.aQute.bndlib/src/aQute/bnd/osgi/AnnotationHeaders.java

```diff
@@ -138,6 +155,14 @@
 	// we parse the annotations separately at the end
 	boolean							finalizing;
 
+	static String convert(Object value) {
+		try {
+			return CONVERTER.convert(String.class, value);
```

Comment:
The doc will also need to say how each element type is converted. E.g. Enum to a String of the enum name. (Hence the need to `${tolower}` in the test case above). It is unclear how arrays are handled here, or annotation values. We probably will need tests for the handling of all the element types (not just string and enum).

Reply:
yup, good point